reco_utils/recommender/surprise/surprise_utils.py

```diff
@@ -0,0 +1,49 @@
+import numpy as np
+import pandas as pd
+
+from reco_utils.common.constants import DEFAULT_USER_COL, DEFAULT_ITEM_COL, DEFAULT_RATING_COL
+
+
+def compute_predictions(algo, data, usercol=DEFAULT_USER_COL, itemcol=DEFAULT_ITEM_COL):
```

Comment:
can we try to find a better name for `compute_predictions` and `compute_all_predictions`, the first one is for rating, in SAR for example we call that predict, and the second one for ranking. In SAR we call this recommend_k_items. This leads to the second comment below

Reply:
`compute_rating_predictions`, `compute_ranking_predictions`?

Reply:
sounds good to me